contrib/dashboards/jsonnet/dashboard_details.jsonnet

```diff
@@ -15,6 +15,22 @@ dashboard.new(
   time_from="now-15m",
   tags=['fl2m','grafana','dashboard','details'],
 )
+.addPanel(
+  graphPanel.new(
+    datasource='prometheus',
+    title="Bandwidth per src and destination subnet",
+  )
+  .addTarget(
+    prometheus.target(
+      expr='topk(10,rate(fl2m_bandwidthPerSourceDestinationSubnet[1m]))',
```

Comment:
add underscores between words like in https://github.com/netobserv/flowlogs2metrics/blob/7571d858e299d18966d99bebebf4ce720f1ca203/contrib/dashboards/jsonnet/dashboard_details.jsonnet#L41

This need to be updated for all place bellow :-)

Reply:
Done